system/lib/pthread/emscripten_futex_wait.c

```diff
@@ -116,7 +116,7 @@ int emscripten_futex_wait(volatile void *addr, uint32_t val, double max_wait_ms)
     return -EINVAL;
   }
 
-  _emscripten_yield();
+  _emscripten_yield(0);
```

Comment:
Is the meaning of `now=0` documented anywhere?

Reply:
This is an internal API and is not documented anywhere.   Nobody should be calling it directly, it should only be used internally.

I did notice that its declared in `emscripten/threading.h` rather than `threading_internal.h` which is shame.. we should probably move it.    But at least it has the underscore prefix that marks it as internal.

Reply:
Actually this change already moves the definition in `emscripten/threading.h`.. i had forgetting I had done that.

Reply:
Added comment regarding the `now=0` behaviour.